docs/admin/logs/_index.md

```diff
@@ -0,0 +1,176 @@
+
+Setting up log collection requires two steps:
+
+1. Running a log forwarding DaemonSet on each node.
+2. Running a collector somewhere in the cluster.
+
+In the following example, a StatefulSet is used, which stores logs on a Kubernetes PersistentVolumeClaim, but you can also use a HostPath.
+
+## Setting up the collector
+
+Set up the collector before the forwarders. You will need the address of the collector when configuring the forwarders, and the forwarders may queue logs until the collector is ready.
+
+![System diagram: forwarders and co-located collector and nginx](system.svg)
+
+<!-- yuml.me UML rendering of:
+[Forwarder1]logs->[Collector]
+[Forwarder2]logs->[Collector]
+
+// Add notes
+[Collector]->[shared volume]
+[nginx]-[shared volume]
+-->
+
+The [`fluent-bit-collector.yaml`](./fluent-bit-collector.yaml) defines a
+StatefulSet, as well as a Kubernetes Service which allows accessing and reading
+the logs from within the cluster. The supplied configuration will create the
+monitoring configuration in a namespace called `logging`.
+
+You can apply the configuration by entering the command:
+
+```shell
+kubectl apply --filename https://github.com/knative/docs/raw/main/docs/install/collecting-logs/fluent-bit-collector.yaml
+```
+
+The default configuration will classify logs into:
+
+- Knative apps, or pods with an `app=Knative` label
+- Non-Knative apps
+
+The logs default to logging with the pod name; this can be changed by updating the `log-collector-config` ConfigMap before or after installation. Once the ConfigMap is updated, you must restart Fluent Bit. You can do this by deleting the pod and letting the StatefulSet recreate it.
+
+To access the logs through your web browser, enter the command:
+
+```shell
+kubectl port-forward --namespace logging service/log-collector 8080:80
+```
+
+And then visit http://localhost:8080/.
+
+You can also open a shell in the `nginx` pod and search the logs using unix
+tools, by entering the command:
+
+```shell
+kubectl exec --namespace logging --stdin --tty --container nginx log-collector-0
+```
+
+## Setting up the forwarders
+
+Follow the [Fluent Bit directions for installing on Kubernetes](https://docs.fluentbit.io/manual/installation/kubernetes) documentation to set up a Fluent Bit DaemonSet which forwards logs to ElasticSearch by default.
+
+When the directions call for creating the ConfigMap, you need to replace the ElasticSearch configuration with the [`fluent-bit-configmap.yaml`](./fluent-bit-configmap.yaml), or add the following block to the ConfigMap, and update the
+`@INCLUDE output-elasticsearch.conf` to be `@INCLUDE output-forward.conf`.
+
+```yaml
+output-forward.conf: |
+  [OUTPUT]
+      Name            forward
+      Host            log-collector.logging
+      Port            24224
+      Require_ack_response  True
+```
+
+## Local collector
+
+**NOTE:** This describes a development environment setup, and is not appropriate
```

Comment:
In mkDocs, this would be a great candidate for: 

```
!!! warning
        This describes a development environment setup, ... 
```
        
It would render like this: 

<img width="689" alt="Screen Shot 2021-05-05 at 10 32 01 AM" src="https://user-images.githubusercontent.com/16281246/117158133-26ed1e00-ad8d-11eb-8581-94de2ef35d54.png">

Reply:
Can you put all this in a guide please instead of on the PR 😄 I'm not gonna be implementing it here yet